aikau/src/main/resources/alfresco/forms/controls/BaseFormControl.js

```diff
@@ -303,6 +304,22 @@ define(["dojo/_base/declare",
       getPubSubOptionsImmediately: true,
 
       /**
+       * This attribute should only be configured to be true when the control is being used within the
+       * a [list]{@link module:alfresco/lists/AlfList} - for example as part of the widget model for
+       * [a list appendix]{@link module:alfresco/lists/views/AlfListView#widgetsForAppendix} or as 
+       * part of the rendering of an item. Configuring this to true will ensure that using the cursor
+       * keys will not control the selected list item and that focus will not be stolen from the wrapped
+       * control - however, this should be used with caution as it may limit or alter the behaviour
+       * of some form controls. See AKU-920/AKU-921 for the background of why this has been added.
+       * 
+       * @instance
+       * @type {boolean}
+       * @default
+       * @since 1.0.63
+       */
+      usedInList: false,
```

Comment:
This feels horribly like "coding by exception". Is this definitely the best way to fix the problem?